ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -3950,6 +3951,8 @@ static void ggml_vk_load_shaders(vk_device& device) {
 
     ggml_vk_create_pipeline(device, device->pipeline_sum_rows_f32, "sum_rows_f32", sum_rows_f32_len, sum_rows_f32_data, "main", 2, sizeof(vk_op_sum_rows_push_constants), {1, 1, 1}, { device->subgroup_size }, 1);
 
+    ggml_vk_create_pipeline(device, device->pipeline_cumsum_f32, "cumsum_f32", cumsum_f32_len, cumsum_f32_data, "main", 2, sizeof(vk_op_sum_rows_push_constants), {1, 1, 1}, { 128, device->subgroup_size }, 1, true, true, device->subgroup_size);
```

Comment:
Is it necessary to force the subgroup size? That's usually not good for performance on Intel, at least if you force it to 32.

Reply:
Yeah, I see that's required for the shared memory size. I think we can take another look at it once we come across it in a model.